deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-vertx/pom.xml

```diff
@@ -619,4 +404,18 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
-</project>
+
+    <build>
```

Comment:
Wondering if we could use pluginManagement here instead? (Same with the other compiler declarations)

Reply:
Done